src/main/java/competition/subsystems/armlifting/commands/LiftingArmCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.armlifting.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.armlifting.LiftingSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class LiftingArmCommand extends BaseCommand {
+
+    final LiftingSubsystem liftingSubsystem;
+    final OperatorInterface oi;
+
+    @Inject
+    public LiftingArmCommand(OperatorInterface oi, LiftingSubsystem liftingSubsystem) {
+        this.oi = oi;
+        this.liftingSubsystem = liftingSubsystem;
+        this.requires(this.liftingSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        liftingSubsystem.up();
+        liftingSubsystem.down();
```

Comment:
Like before, this will make the robot go up then down, very very quickly. You should only do one or the other.

Reply:
I decided to keep liftingSubsystem.up and removed .down.